src/treasuryDAO/types.mligo

```diff
@@ -29,3 +29,21 @@ type treasury_contract_extra =
   * tez // min_xtz_amount
   * tez // max_xtz_amount
   )
+
+type normal_proposal =
```

Comment:
This is the same as the `transfer_type` in registryDAO.
So I think its definition could be moved to `src/common/types.mligo` instead.

Reply:
Yeah, they are actually the same, I've made this change, as well as other changes, should be good for re-review.